rust/actyx/ax/src/cmd/users/devcert.rs

```diff
@@ -0,0 +1,48 @@
+use crate::{
+    certs::{AppDomain, DeveloperCertificateInput, ManifestDeveloperCertificate},
+    cmd::AxCliCommand,
+    crypto::{PrivateKey, PublicKey},
+    util::formats::{ActyxOSCode, ActyxOSResult, ActyxOSResultExt},
+};
+use futures::{stream::once, FutureExt, Stream};
+use structopt::StructOpt;
+
+#[derive(StructOpt, Debug)]
+#[structopt(version = crate::util::version::VERSION.as_str())]
+pub struct DevCertOpts {
+    /// The secret key used to sign the certificate
+    /// (this must match the AX_PUBLIC_KEY your `actyx` binary has been compiled with).
+    #[structopt(long, short = "A", env, hide_env_values = true)]
+    ax_secret_key: PrivateKey,
+
+    /// The developer's public key.
+    #[structopt(long, short)]
```

Comment:
probable breaking change: this previously has `env` attribute

Reply:
It isn't breaking because it was only used by us and it is only exposed now